igcollect/logfile_values.py

```diff
@@ -50,7 +50,7 @@ def __init__(self, arg):
         self.period = period
         self.values = []  # Container for metric values
         self.last_value = 0
-        self.now = int(datetime.datetime.utcnow().timestamp())
+        self.now = int(datetime.now(timezone.utc).timestamp())
```

Comment:
Is the specification for utc necessary anyway? Shouldn't `timestamp` already give you the "utc" timestamp? Different timezones doesn't change the timestamp, no?

Reply:
I'm not sure, just following the documentation and trying to leave the desired behavior. Unfortunately, right now it's just broken =\

Reply:
`flake8 igcollect/logfile_values.py` keeps silence

Reply:
Not sure what `flake8` has to do with that. Made a quick test:
```
>>> print(int(datetime.datetime.now(datetime.timezone.utc).timestamp()))
1587572110
>>> print(int(datetime.datetime.now().timestamp()))
1587572111
```
It's the same. `utcnow` is different though, that's correct. So the fix is correct, but `datetime.timezone.utc` is not necessary as parameter.

Reply:
I partially agree, but that's the statement of the utcnow() documentation

```
An aware current UTC datetime can be obtained by calling datetime.now(timezone.utc)
```

And as I like to simplify everything, I just don't like kidding with time =)